js/index.js

```diff
@@ -1,273 +0,0 @@
-let windowW = window.innerWidth;
-let windowH = window.innerHeight;
-let bgColor = "lightgray";
-let jack;
-let enemies;
-let horizontalLine;
-let lineGap = windowH / 8;
-let holeW = lineGap;
-let jumpingTime;
-
-const ironHackBlue = "#009bff";
-
-let gameSpeed = 4;
-let score;
-let gameTime;
-let bonusTime = 10;
-
-let spaceBarKey = 32;
-let gameState = "playing";
-
-let bgMusic;
-let jumpSound;
-let winSound;
-let loseSound;
-let moveSound;
-let gameMusic = "OFF";
-
-function preload() {
-  jackImg = loadImage("assets/images/jhack.png");
-  jackImgLeft = loadImage("assets/images/jhack_left.png");
-  jackImgRight = loadImage("assets/images/jhack_right.png");
-  gameMusicImg = loadImage("assets/images/extra.png");
-  bgMusic = loadSound("assets/sounds/bgMusic.mp3");
-  jumpSound = loadSound("assets/sounds/jump.mp3");
-  fallSound = loadSound("assets/sounds/fall1.mp3");
-  winSound = loadSound("assets/sounds/win.wav");
-  loseSound = loadSound("assets/sounds/lose.wav");
-  moveSound = loadSound("assets/sounds/move.mp3");
-  fallSound = loadSound("assets/sounds/fall1.mp3");
-}
-
-function setup() {
-  const canvas = createCanvas(windowW, windowH);
-  canvas.parent("game-board");
-  bgMusic.setVolume(0.2);
-  initializeGame();
-  soundFormats("mp3", "wav");
-  noLoop();
-}
-
-function draw() {
-  if (gameState === "playing") {
-    playGame();
-    return;
-  }
-  if (gameState === "gameover") {
-    gameOver();
-  }
-  if (gameState === "win") {
```

Comment:
for these three functions it's cool with the if blocks but you could imagine something like this, eventually becoming useful: 
```js
const stateToFunctionMap = {
   playing: playGame,
   gameOver: gameOver,
   win: win, 
}
```
because then draw would become this: 

```js
stateToFunctionMap[gameState]
```